folium/map.py

```diff
@@ -22,7 +22,7 @@
 
 _default_js = [
     ('leaflet',
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"),
+     "https://npmcdn.com/leaflet@1.0.0-rc.3/dist/leaflet.js"),
```

Comment:
Thanks! I will take a looking into testing the latest `leaflet` later. I will reserve this weekend for `folium`.